atlas/schema.hcl

```diff
@@ -1,11 +1,492 @@
-table "test" {
+table "expense_snapshot_v1" {
   schema = schema.public
-  column "id" {
+  comment = "A snapshot in time of a given expense entry. A new snapshot is created every time the expense entry is modified."
+  column "expense_id" {
     null = false
-    type = integer
+    type = bigint
+  }
+  column "snapshot_id" {
+    null = false
+    type = bigint
+    identity {
+      generated = ALWAYS
+      start = 0
+      increment = 1
+    }
+  }
+  column "scan_id" {
+    null = true
+    type = bigint
+  }
+  column "title" {
+    null = false
+    type = text
+    comment = "The name of item/service expense."
+  }
+  column "amount" {
+    null = false
+    type = money
+    comment = "The amount listed on the receipt for this entry."
+  }
+  column "deductible" {
+    null = false
+    type = double_precision
+    comment = "The percentage of the expense that is deductible."
```

Comment:
What does deductible mean in the context of taxes?

Reply:
If an expense is deductible, then it can be subtracted from taxable income so that you get taxed less.

https://www.investopedia.com/articles/tax/09/self-employed-tax-deductions.asp

Reply:
nice can you put this in the comment 🥇

Reply:
please put in comment

Reply:
no <3

Reply:
bump